Sources/Tracking/migration/DataPipelineMigrationAssistant.swift

```diff
@@ -2,14 +2,21 @@ import CioDataPipelines
 import CioInternalCommon
 import Foundation
 
+public protocol DataPipelineMigration: AutoMockable {
+    func handleAlreadyIdentifiedMigratedUser()
+    func handleQueueBacklog()
+    func getAndProcessTask(for task: QueueTaskMetadata)
```

Comment:
```suggestion
```

Nit-picky, but this function feels like an internal/private function only needed inside of `DataPipelineMigrationAssistant` implementation. So, not sure why we need to expose it in a protocol.